compiler/luci/tests/test.lst

```diff
@@ -160,6 +160,7 @@ addread(Net_ZeroDim_001)
 # from res/CircleRecipes
 addread(BatchMatMul_000)
 addread(InstanceNorm_000)
+addread(MatMul_000)
```

Comment:
This is for the operators created by `circlechef`. You can just add to above line like `BatchMatMulV2`.